### ISSUE TITLE
eventContent moves arrow event length indicators

### ISSUE THREAD

**Issue by CraftyDlx** [fullcalendar/fullcalendar#5547]
I am using v5-rc1 and added some extra info to my event titles using 
```
eventContent: function(info) {
  return info.event.title + " (" + info.event.extendedProps.eventId +")";
},
```
I understand that the small arrows left and right of an event bar indicate that the event is longer than the actual view. That's how it should look like:
![Screen Shot 06-25-20 at 10 49 PM](https://user-images.githubusercontent.com/59885175/85794422-e79a7d80-b736-11ea-8b1f-621fcd2c94bb.PNG)

After using eventContent the right arrow slides from the right border to the end of the title:
![Screen Shot 06-25-20 at 10 48 PM](https://user-images.githubusercontent.com/59885175/85794394-d8b3cb00-b736-11ea-8841-53ba49718b82.PNG)

This is not intentional, is it? Or have the arrows another meaning?

There is one more thing: Is it possible to use eventContent only for one event feed when there are more than one? I tried
```
      eventSources: [
          {
            url: 'http://localhost/matpro/public/event-feed',
            eventContent: function(info) {
              return info.event.title + " (" + info.event.extendedProps.eventId +")";
            },
          },
          {
            url: 'http://localhost/matpro/public/holiday-feed',
            backgroundColor: 'coral',
            textColor: 'black',
            display: 'background'
          }
      ],
```
but this way it does not work at all.

**Comment by acerix** [comment]
Thanks, confirmed.

https://codepen.io/acerix/pen/xxZreGX?editors=001

You can check the id of the event's source in `eventContent` and not return anything for those sources which will use the default content.

**Comment by arshaw** [comment]
fixed in v5.1.0

https://codepen.io/arshaw/pen/OJMxbER?editors=001
